tests/test_browser.py

```diff
@@ -73,6 +73,25 @@ def shell_with_script(shell_file, output_file, replacement):
       output.write(input.read().replace('{{{ SCRIPT }}}', replacement))
 
 
+def is_chrome():
+  return EMTEST_BROWSER and 'chrom' in EMTEST_BROWSER.lower()
```

Comment:
why without the `e`?

Reply:
I think its to support "chromium" , this line of code is not new, I just moved it.